python/amici/petab_objective.py

```diff
@@ -343,6 +344,17 @@ def _set_initial_concentration(condition_id, species_id, init_par_id,
                                        par_map, scale_map):
             value = petab.to_float_if_float(
                 petab_problem.condition_df.loc[condition_id, species_id])
+            if pd.isna(value):
+                value = float(
+                    get_species_initial(
+                        petab_problem.sbml_model.getSpecies(species_id)
+                    )
+                )
+                logger.warning(f'The species {species_id} has no initial value'
```

Comment:
Not sure I'd go for warnings, as this is explicitly a feature in PEtab. Might be confusing. Maybe change to info?

Reply:
Ah, maybe not worth logging then. Switched to debug but could also remove.

Reply:
I think debug can be helpful